### ISSUE TITLE
AttributeError at ".../jsi18n"

### ISSUE THREAD

**Issue by jedie** [django-treebeard/django-treebeard#90]
I have the same issues with django 1.11 as in #88

I'm sorry, I can't tell you exactly, but I think it workes with django 1.8 in the past, but not with django 1.11...

`<script type="text/javascript" src="jsi18n"></script>`
will "resolve" into url: `/<lang_code/admin/<app_name>/<model_name>/jsi18n/` and this url failed with:

```
...
File "lib/python3.5/site-packages/django/views/generic/base.py" in dispatch
  88.         return handler(request, *args, **kwargs)

File "lib/python3.5/site-packages/django/views/i18n.py" in get
  360.         packages = packages.split('+') if packages else self.packages

Exception Type: AttributeError at /de/admin/product/productcategory/jsi18n/
Exception Value: 'tuple' object has no attribute 'split'
```
The `packages` variable is: `('treebeard',)`

(pull request is on the way)

**Comment by jedie** [comment]
Is it predictable when the new version >4.2.0 with the fix will be released?

**Comment by Senpos** [comment]
Yep, i wish i could grab it from PyPi.

Using `treebeard` with `Django 2.0.3` on `Python 3.6.4` and have this issue.

**Comment by jrief** [comment]
Today I released version 4.2.1 which fixes this.
